chaoslib/litmus/platform/gke/node-cpu-hog-job.j2

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Job
+metadata:
+  name: node-cpu-hog
+  labels: 
+    chaosUID: {{ chaos_uid }}
+spec:
+  template:
+    metadata:
+      labels:
+        app: node-cpu-hog
+        chaosUID: {{ chaos_uid }}
+    spec:
+      nodeSelector:
+        kubernetes.io/hostname: {{ node_name }}
+      restartPolicy: Never
+      containers:
+      - image: litmuschaos/cpu:latest
+        imagePullPolicy: Always
+        name: node-cpu-hog
+        command: ["/bin/bash", "stress", "--cpu", "{{ cpu_limit }}", "--timeout", "{{ c_duration }}"]
+        args: ["-c", "sleep 10000"]
```

Comment:
Why are you putting sleep if you already have timeout var in command?

Reply:
WAIT THE PR IS IN WIP